openclsim/model.py

```diff
@@ -302,6 +314,16 @@ def single_run_process(
             amount, mover.check_optimal_filling(loader, unloader, origin, destination)
         )
 
+    if (
+        isinstance(mover, core.Routeable)
+        and mover.optimize_route == True
+        and type(mover.loadfactors) != type(None)
+    ):
+        opt_LF = mover.check_optimal_filling_Roadmap(
```

Comment:
maybe add a comment or increase the length of variable opt_LF so it is clear what it returns

Reply:
done